oeps/oep-0002.rst

```diff
@@ -47,7 +47,7 @@ Specification
 Each repo in the Open edX system will include a file ``openedx.yaml``, with the
```

Comment:
Yes, https://github.com/edx/repo-tools/tree/master/edx_repo_tools/oep2/checks has all of the automatic checkers that we've written for oeps.